src/clj/rems/db/test_data.clj

```diff
@@ -409,10 +437,11 @@
     (create-applications! simple (:simple workflows) (+demo-users+ :applicant1) (+demo-users+ :approver1))
     (create-disabled-applications! disabled (:simple workflows) (+demo-users+ :applicant1) (+demo-users+ :approver1))
     (create-bundled-application! simple bundlable (:simple workflows) (+demo-users+ :applicant2) (+demo-users+ :approver1))
-    (create-review-application! with-review (:with-review workflows) +demo-users+)
+    (create-review-applications! with-review (:with-review workflows) +demo-users+)
     (create-application-with-expired-resource-license! (:simple workflows) form +demo-users+)
     (create-application-before-new-resource-license! (:simple workflows) form +demo-users+)
     (create-expired-license!)
     (let [dynamic (create-catalogue-item! res1 (:dynamic workflows) form
                                           {"en" "Dynamic workflow" "fi" "Dynaaminen työvuo"})]
-      (create-dynamic-application! dynamic (:dynamic workflows) (+demo-users+ :applicant1)))))
+      (create-dynamic-application! dynamic (:dynamic workflows) (+demo-users+ :applicant1))
+      (create-dynamic-request-applications! dynamic (:dynamic workflows) +demo-users+))))
```

Comment:
hmm why dynamic-request-application vs dynamic-application

also should these functions just be combined?

Reply:
They sure can be. The other tries to create just different kinds of requests but I think I'll combine them.